src/xcode/ENA/ENA/Source/Developer Menu/Features/DMWarnOthersNotificationViewController.swift

```diff
@@ -0,0 +1,198 @@
+	}
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		view.backgroundColor = .systemBackground
+		
+		let titleLabel = UILabel(frame: .zero)
+		titleLabel.translatesAutoresizingMaskIntoConstraints = false
+		titleLabel.numberOfLines = 0
+		titleLabel.text = "Warn others notification settings"
+		titleLabel.font = UIFont.enaFont(for: .headline)
+		
+		let descriptionLabel = UILabel(frame: .zero)
+		descriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+		descriptionLabel.numberOfLines = 0
+		descriptionLabel.text = "You can use the two input fields below to maintain the time until the notifications will be shown.\nTime is maintained in seconds."
+		descriptionLabel.font = UIFont.enaFont(for: .subheadline)
+		
+		let saveButton = UIButton(frame: .zero)
+		saveButton.translatesAutoresizingMaskIntoConstraints = false
+		saveButton.setTitle("Save timer values", for: .normal)
+		saveButton.addTarget(self, action: #selector(scheduleNotificationsButtonTapped), for: .touchUpInside)
+		saveButton.setTitleColor(.enaColor(for: .buttonPrimary), for: .normal)
+		
+		let resetDefaultsButton = UIButton(frame: .zero)
+		resetDefaultsButton.translatesAutoresizingMaskIntoConstraints = false
+		resetDefaultsButton.setTitle("Reset values to default", for: .normal)
+		resetDefaultsButton.addTarget(self, action: #selector(resetDefaultsButtonTapped), for: .touchUpInside)
+		resetDefaultsButton.setTitleColor(.enaColor(for: .buttonDestructive), for: .normal)
+		
+		let descriptionLabelResetNotifications = UILabel(frame: .zero)
+		descriptionLabelResetNotifications.translatesAutoresizingMaskIntoConstraints = false
+		descriptionLabelResetNotifications.numberOfLines = 0
+		descriptionLabelResetNotifications.text = "If you want to let the notifications get scheduled again (eg. for testing purposes), just press the reset button below and after that you navigate back to the corresponding views of the app, the notifications will get scheduled again.\nYou can also check 'DM Pending Notifications', to check which ones are scheduled."
+		descriptionLabelResetNotifications.font = UIFont.enaFont(for: .subheadline)
+		
+		let resetNotificationsButton = UIButton(frame: .zero)
+		resetNotificationsButton.translatesAutoresizingMaskIntoConstraints = false
+		resetNotificationsButton.setTitle("Reset notification state", for: .normal)
+		resetNotificationsButton.addTarget(self, action: #selector(resetNotificationsButtonTapped), for: .touchUpInside)
+		resetNotificationsButton.setTitleColor(.enaColor(for: .buttonDestructive), for: .normal)
+		
+		time1Label = UILabel(frame: .zero)
+		time1Label.translatesAutoresizingMaskIntoConstraints = false
+		time1Label.numberOfLines = 0
+		time1Label.text = "Notification 1 time (sec)"
+		
+		time2Label = UILabel(frame: .zero)
+		time2Label.translatesAutoresizingMaskIntoConstraints = false
+		time2Label.numberOfLines = 0
+		time2Label.text = "Notification 2 time (sec)"
+		
+		time1Textfield = UITextField(frame: .zero)
```

Comment:
`TextField` with a capital F, goes for both text fields

Reply:
Done